### PR TITLE
Airtable support for PNI products

### LINKED ISSUE #6008: PNI to Airtable connection
We want to push Wagtail-based PNI products to an Airtable sheet. We'll want to use the [wagtail-airtable](https://github.com/wagtail/wagtail-airtable) package. 

Some things we'll need to know:
- [ ] Is this a one-way or two-way data sync? As in, can we export data from Wagtail to Airtable and vice versa (two-way)? Or Only send data to Airtable from Wagtail (one-way)?
- [ ] What kind of data do we want to store in Airtable? We can add just about everything except images. We probably want an exact product URL, product name, product category or categories, and what else would the Airtable users be looking for? 
- [ ] It'll be easiest if I have Airtable access to I can setup the Airtable columns (for things like dropdowns, enums, checkboxes, etc). We will want to look into that. Though I can always walk someone through it if permissions are going to cause problems. 
- [ ] While we're adding Airtable support for PNI products, are there any other models or data types we want to automagically push to Airtable? (If there are, we should consider opening a small epic).

CC @benhohner

### PR DESCRIPTION
**Pull request by KalobTaulien**
Closes #6008 

**Link to sample test page**: https://foundation-s-6008-pni-t-mwlsud.herokuapp.com/cms/pages/104/edit/

**Sample Airtable sheet**: https://airtable.com/tblDv7WCDL2xNZAtn/viwxQjDZuERYodSs1?blocks=hide

**How to test:**
1. Edit any PNI product (publish or save draft, either one is fine)
2. Open the airtable link from above, and your PNI product should be in there.
3. Update the PNI product in Wagtail
4. Check it updates in Airtable with your new data. 

**Notes**:
Both General and Software products are in one large Airtable sheet. 
We need environment vars added to Production including:
- [x] AIRTABLE_API_KEY
- [x] AIRTABLE_BASE_KEY
- [x] AIRTABLE_BASE_URL
- [x] AIRTABLE_ENABLED (set tot True for Production, and False for review apps; after this PR is merged in)

### CODE REVIEW

File: network-api/networkapi/wagtailpages/pagemodels/products.py [hunk 1]
```diff
@@ -359,6 +361,59 @@ class ProductPage(FoundationMetadataPageMixin, Page):
...
+            "Strong password": self.strong_password,
+            "Strong password help text": self.strong_password_helptext,
+            "Manages security vulnerabilities": self.manage_vulnerabilities,
+            "Manages security help text": self.manage_vulnerabilities_helptext,
+            "Has privacy policy": self.privacy_policy,
+            "Privacy policy help text": self.privacy_policy_helptext,
+            "Phone number": self.phone_number,
+            "Live chat": self.live_chat,
+            "Email address": self.email,
+            "Twitter": f"https://twitter.com/{self.twitter}" if self.twitter else ''
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/pagemodels/products.py:407]: hot damn

**Review by Pomax — APPROVED** [review]
this looks fairly sane to me

File: network-api/networkapi/wagtailpages/pagemodels/products.py (hunk 1 above)
Comment (KalobTaulien) [reply network-api/networkapi/wagtailpages/pagemodels/products.py:407]: Right? So many fields!

**Review by Pomax — APPROVED** [review]
still looks sane

**Review by Pomax — APPROVED** [review]
pirate.gif

**Pull request merged**